src/Raven.Studio/typescript/viewmodels/resources/widgets/ongoingTasksWidget.ts

```diff
@@ -0,0 +1,176 @@
+import clusterDashboard = require("viewmodels/resources/clusterDashboard");
+import websocketBasedWidget = require("viewmodels/resources/widgets/websocketBasedWidget");
+import clusterTopologyManager = require("common/shell/clusterTopologyManager");
+
+enum taskName {
+    ExternalReplication = 0,
+    ReplicationHub,
+    ReplicationSink,
+    RavenEtl,
+    OlapEtl,
+    SqlEtl,
+    Backup,
+    Subscription
+}
+
+class ongoingTaskItem {
+    protected clusterManager = clusterTopologyManager.default;
+    
+    taskName = ko.observable<taskName>();
+    taskCount = ko.observable<number>();
+    
+    taskNodes = ko.observable<Set<string>>();
```

Comment:
do we need set here? It doesn't work well in views and knockout.

Reply:
Leaving this as discussed